README.md

```diff
@@ -128,6 +128,10 @@ in various programming languages.
 * **multicont**: continuations with multi-shot semantics in OCaml  
   by Daniel Hillerström  
   ([www](https://github.com/dhil/ocaml-multicont))
+  
+* **Freak**: a programming language with coalgebraic effects and cohandlers  
```

Comment:
Please move this entry up to the capitalised names in such a way that the list remains alphabetically sorted.

Reply:
@dhil done, inserted in alphabetical order